src/util/psbt/error.rs

```diff
@@ -37,8 +37,12 @@ pub enum Error {
     /// Magic bytes for a PSBT must be the ASCII for "psbt" serialized in most
     /// significant byte order.
     InvalidMagic,
+    ///Missing both the witness and non-witness utxo
+    MissingUtxo,
     /// The separator for a PSBT must be `0xff`.
     InvalidSeparator,
+    /// Triggered when output index goes out of bounds of the output in non witness utxo
```

Comment:
```suggestion
    /// Triggered when output index does not exist in PSBT input (non-witness) UTXO.
```

Not sure if the content of this comment is correct but it shows a grammatical improvement, please modify as you see fit.

Reply:
While strictly speaking correct, thinking of it in terms of "out of bounds" feels more natural to me. Won't block if others prefer your suggestion.

Reply:
Fair comment, this is a tricky little rustdoc to write succinctly.

Reply:
out of bound seemed natural to me too, but whatever you guys prefer at the end though.

Reply:
How's this?
```
    /// Triggered when output index is out of bounds in relation to the output in non-witness UTXO.
```

Reply:
Sounds good. But should it be "triggered"? Maybe "returned is better"?

Reply:
I think in case of error words like triggered, panic, etc. Sounds more suitable than "returned", don't you think ?

Reply:
My mental model in Rust is that errors are returned (well, they actually are :)). But honestly, I don't care much if others find "triggered" better.

Reply:
FWIW we have 'triggered' in one other place in the repo
```
git grep trig                                                                                                                                                              
src/blockdata/script.rs:1279:        // which triggerred overflow bugs on 32-bit machines in script formatting in the past.
```

I'm not fussed either way.